corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -224,6 +215,11 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
 
     Views.CaseTileView = Views.CaseView.extend({
         template: "#case-tile-view-item-template",
+        templateHelpers: function () {
+            var dict = Views.CaseTileView.__super__.templateHelpers.apply(this, arguments);
```

Comment:
I've been using this in this file for a while IE https://github.com/dimagi/commcare-hq/blob/3430355aa1e6ba40a38968dba3536dcb9aaef2c5/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L315-L315

So I guess relatively supported?